README.rst

```diff
@@ -113,7 +113,7 @@ Other command-line options::
                             (default)
       --include-archived    include archived repositories
       --exclude-archived    exclude archived repositories (default)
-      --include-private     include private repositories (default)
+      --include-private     include private repositories, requires a github token (default)
```

Comment:
How about 

```suggestion
      --include-private     include private repositories (default when a github token is provided)
```

instead?  Your suggestion makes me think a github token is always required, unless you explicitly --exclude-private.